RadixWallet/Clients/AccountPortfoliosClient/AccountPortfoliosClient+Live.swift

```diff
@@ -177,15 +178,27 @@ extension AccountPortfoliosClient: DependencyKey {
 			return portfolio
 		}
 
+		@Sendable
+		func syncAccountsDeletedOnLedger() {
+			Task.detached {
+				let isDeletedAccountDetected = try? await SargonOS.shared.syncAccountsDeletedOnLedger()
+				if isDeletedAccountDetected == true {
+					overlayWindowClient.scheduleAlertAndIgnoreAction(.deletedAccountDetectedAlert)
+				}
+			}
+		}
+
 		return AccountPortfoliosClient(
 			fetchAccountPortfolios: { accountAddresses, forceRefresh in
 				try await Task.detached {
-					try await fetchAccountPortfolios(accountAddresses, forceRefreshEntities: forceRefresh, forceRefreshPrices: forceRefresh)
+					syncAccountsDeletedOnLedger()
```

Comment:
`syncAccountsDeletedOnLedger` is being called from this Task which already detached, and then `syncAccountsDeletedOnLedger` itself uses `Task.detached`, hmm, do we do that because we **want** `fetchAccountPortfolios` (called below) to be performed in parallell? because that is what will happen right? `fetchAccountPortfolios` can be called - and finished - before `syncAccountsDeletedOnLedger`  is done. Is that the intention?

Otherwise since `syncAccountsDeletedOnLedger` is **only** called in two already detached and async contexts I think maybe it (`syncAccountsDeletedOnLedger`) can be marked async? and removed its Task.detached.

I'm not completely sure though. I just to explore the possibility of this working "by chance" and see if we should try to use more structured async. 

We have the `State` value, on which we can have `Task<..>?` as well and cancel any previous task and replace with new ones. 

e.g.

```diff 
actor State {
...
+ var deletedAccountsSync: Tas<...>?
...
}
```

```diff
static let liveValue: AccountPortfoliosClient {
    let state = State()
...
    @Sendable func syncAccountsDeletedOnLedger() {
+        state.deletedAccountsSync.cancel()
+        state.deletedAccountsSync = nil
+        state,deletedAccountsSync = Task { ...}
    }
...
}
```

Reply:
Yes, the intention was to perform them in parallel, but it’s not really necessary. I can make `syncAccountsDeletedOnLedger` async and remove Task